dev/ToastNotifications/ToastNotificationUtility.cpp

```diff
@@ -188,6 +188,9 @@ CATCH_RETURN()
 
 std::wstring RegisterComActivatorGuidAndAssets(winrt::Microsoft::Windows::ToastNotifications::ToastActivationInfo const& details)
 {
+    std::wstring toastAppId{ RetrieveToastAppId() };
+    THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(toastAppId.c_str(), GUID_NULL));
```

Comment:
Unclear why this was added here and not done seperately. Can you elaborate?

Reply:
Since this is where the ComActivator/Assets are being put in the registry, I put the registration here as well because the toastAppId is being generated here. We can pull this out of this function and put it in RegisterActivator. What do you think?

Reply:
Yes. I think we should do that and break it up. This is specifically for the COM activator and Nitro scenario in my interpretation.